### ISSUE TITLE
Kafka message to IBM MQ on mainframe

### ISSUE THREAD

**Issue by vivekanandak** [ibm-messaging/kafka-connect-mq-source#4]
Hi,
Is there a specific instructions for connecting Kafka messages to MQ on mainframe?

**Comment by AndrewJSchofield** [comment]
I have confirmed that this works. I have successfully transferred messages from MQ on z/OS v9 to Kafka using this connector.

**Comment by vivekanandak** [comment]
Hi,

Thanks for the response. Could you please share the code. It really helps.
Does any access restrictions are there to get the access from z/os to kafka?

Thanks,
Vivek

**Comment by AndrewJSchofield** [comment]
There is no code to share. I just configured the SVRCONN channel on the z queue manager, put the hostname in the configuration for the connector and started it up. I used a pretty simple configuration:

value.converter=org.apache.kafka.connect.storage.StringConverter
mq.message.body.jms=true
mq.record.builder=com.ibm.mq.kafkaconnect.builders.DefaultRecordBuilder

Then it just worked. Good luck.
